proposals/privacy.md

```diff
@@ -132,14 +136,18 @@ links (identified by relation type);
 metadata (creation time, modification time, support, and version information);
 data schemas;
 and URI templates.
+ML: This mechanism is not part of the current specs.
```

Comment:
Correct.  We are saying that any mechanism to deliver TDs (that are associated with an identifiable person...) needs to support these requirements.   But we WILL specify (one possible) mechanism as part of the Discovery spec.